src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,78 @@
+package roomescape.controller;
+
+import java.net.URI;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import roomescape.dto.ReservationRequestDto;
+import roomescape.dto.ReservationResponseDto;
+import roomescape.mapper.DtoMapper;
+import roomescape.model.Reservation;
+import roomescape.utils.RequestValidator;
+
+@Controller
+public class ReservationController {
+
+    private List<Reservation> reservations;

```

Comment:
```suggestion
    private final ReservationRepository reservationRepository;
```

DB가 없어서 Repository를 따로 만들지 않으셨다곤 했지만, Repository를 만들어서 그 객체를 통해 저장/삭제/조회하는 것이 더 좋아보이긴 해요!

Reply:
역할을 분리하는 측면에서 봤을 때, 그것이 훨씬 좋은 방법일 것 같네요. Repository를 구현해서 리팩토링해보겠습니다!

Reply:
코드리뷰 반영하였습니다! e48bef91ad8edd01de45b3451e20094b1d8ed8a7